documentation/staging/content/managing-domains/domain-lifecycle/scaling.md

```diff
@@ -132,17 +176,58 @@ Regardless of which endpoint is being invoked, the URL format for scaling is the
 
 When the operator receives a scaling request, it will:
 
-*	Perform an authentication and authorization check to verify that the specified user is allowed to perform the specified operation on the specified resource.
-*	Validate that the specified domain, identified by `domainUID`, exists.
-*	Validate that the WebLogic cluster, identified by `clusterName`, exists.
-*	Verify that the specified WebLogic cluster has a sufficient number of configured servers or sufficient dynamic cluster size to satisfy the scaling request.
-*	Initiate scaling by setting the `replicas` field within the corresponding Domain, which can be done in either:
-      *	A `cluster` entry, if defined within its cluster list.
-      *	At the domain level, if not defined in a `cluster` entry.
+* Perform an authentication and authorization check to verify that the specified user is allowed to perform the specified operation on the specified resource.
+* Validate that the specified domain, identified by `domainUID`, exists.
+* Validate that the WebLogic cluster, identified by `clusterName`, exists.
+* Verify that the specified WebLogic cluster has a sufficient number of configured servers or sufficient dynamic cluster size to satisfy the scaling request.
+* Verifies that the specified cluster has a corresponding Cluster resource defined or else creates one if necessary.
+* Initiate scaling by setting the `replicas` field within the corresponding Cluster resource.
 
-In response to a change to either `replicas` field, in the Domain, the operator will increase or decrease the number of Managed Server instance Pods to match the desired replica count.
+In response to a change in the `replicas` field in the Cluster resource, the operator will increase or decrease the number of Managed Server instance Pods to match the desired replica count.
 
-#### Using a WLDF policy rule and script action to call the operator's REST scale API
+#### Supported Autoscaling Controllers
+While continuing to support automatic scaling of WebLogic clusters with the WebLogic Diagnostic Framework (WLDF) and Prometheus, Operator 4.0 now supports the [Kubernetes Horizontal Pod Autoscaler (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/).
+##### Kubernetes Horizontal Pod Autoscalar (HPA)
+Automatic scaling of an individual WebLogic cluster, by the Kubernetes Horizontal Pod Autoscalar, is now supported since the Cluster custom resource has enabled the `/scale` subresource. The following step-by-step example illustrates how to configure and run an HPA to scale a WebLogic cluster, `cluster-1`, based on the `cpu utilization` resource metric:
+
+1. Since this example will scale a WebLogic cluster based on CPU utilization, then there is a precondition that the Kubernetes Metrics Server be installed in the cluster:
+```shell
+$ kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
```

Comment:
I've learned that this isn't quite true... In order to scale on "resource metrics" like CPU and memory then the cluster must have installed an implementation of the resource metrics API.  Kubernetes provides one implementation using the Metrics Server that you noted. 

Customers might also want to scale based on "custom metrics". To do that, the customer would have to install an implementation of the custom metrics API. The common use case here is to scale based on metrics from Prometheus. 

Prometheus offers a metrics adapter that implements both the resources and custom metrics API's: https://github.com/kubernetes-sigs/prometheus-adapter. Customers should use this instead if they are using Prometheus for metrics.

*Note*: I don't think that we have time now but we should be able to use this Prometheus adapter and the WebLogic Monitoring Exporter to show autoscaling based on WebLogic specific metrics.

Reply:
For right now, just adjust the wording to say that autoscaling based on resource metrics requires the installation of the Metrics Server or another implementation of the resource metrics API. You could then point out that customers who use Prometheus should consider the Prometheus adapter.